R/utils.R

```diff
@@ -240,6 +240,33 @@ process_text <- function(text,
 
       text <- text %>% escape_latex()
 
+      return(text)
+    }
+  } else if (context == "rtf") {
+
+    # Text processing for LaTeX output
+
+    if (inherits(text, "from_markdown")) {
+
+      text <-
+        text %>%
+        as.character() %>%
+        vapply(commonmark::markdown_html, character(1)) %>%
+        stringr::str_replace_all(c("^<p>" = "", "</p>\n$" = "")) %>%
+        stringr::str_replace_all("<br />\\n|<br />", "<br>")
+
+      return(text)
+
+    } else if (is_html(text)) {
+
+      text <- text %>% as.character()
+
+      return(text)
+
+    } else {
+
+      text <- text
+
```

Comment:
Below (line 272 in the new code), there's an `else` clause that acts like `context == "html"`. We should just throw an error that the context is unknown.